lib/JsonLdDocumentLoader.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+class JsonLdDocumentLoader {
+  constructor() {
+    this.documents = new Map();
+  }
+
+  addStatic(url, document) {
+    this.documents.set(url, document);
```

Comment:
Could add a check that `document` is not `null` and is an object, else throw a `TypeError`.

Reply:
Added validation and tests.